stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/viewholder/message/configurators/IndicatorConfigurator.kt

```diff
@@ -46,7 +46,8 @@ internal class IndicatorConfigurator(
             messageItem.messageReadBy.isNotEmpty() ||
             !messageItem.isMine ||
             message.isInThread() ||
-            message.isEphemeral()
+            message.isEphemeral() ||
+            !style.isReadStateEnabled
```

Comment:
This is the sending / delivered state indicator, are we sure we want to hide this together with the read indicator?

Reply:
I'll better add a new attr for send indicator 👍🏼